Feature/code/ContactDataReaders/CampaignContactDataReader.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System.Linq;
+using Sitecore.Configuration;
+using Sitecore.Data;
+using Sitecore.Diagnostics;
+using Sitecore.EmailCampaign.Model.XConnect.Events;
+using Sitecore.ListManagement.XConnect.Web.Export;
+using Sitecore.XConnect;
+using Sitecore.XConnect.Client;
+using Sitecore.XConnect.Client.Configuration;
+using Feature.Listmanager.xConnectModels;
+
+namespace Feature.Listmanager.ContactDataReaders
+{
+    public class CampaignContactDataReader : IContactDataReader
+    {
+        public string FacetName => ContactCustomFieldsFacet.DefaultFacetKey;
```

Comment:
What's this for?

Reply:
Need to obey IContactDataReader 
`public interface IContactDataReader
    {
        string FacetName { get; }

        string Map(Contact contact);
    }`

Reply:
Hmm I guess the facet doesn't really matter / get used here. I wonder if it can just be empty or just default to Personal Info facet.